sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/windowing/Window.java

```diff
@@ -599,6 +600,33 @@ public void validate(PCollection<T> input) {
       return output.setWindowingStrategyInternal(outputStrategy);
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("windowFn", windowFn.getClass())
+          .include(windowFn);
+
+      if (trigger != null) {
+        builder.add("trigger", trigger.toString());
```

Comment:
Have we verified that the trigger toString is sensible? Do we want to add some kind of "pre-formatted string" type?